packages/miniflare/src/workers/core/entry.worker.ts

```diff
@@ -299,14 +299,22 @@ function colourFromHTTPStatus(status: number): Colorize {
 	return blue;
 }
 
+const ADDITIONAL_RESPONSE_LOG_HEADER_NAME = "X-Mf-Additional-Response-Log";
+
 function maybeLogRequest(
 	req: Request,
 	res: Response,
 	env: Env,
 	ctx: ExecutionContext,
 	startTime: number
-) {
-	if (env[CoreBindings.JSON_LOG_LEVEL] < LogLevel.INFO) return;
+): Response {
+	res = new Response(res.body, res); // Ensure mutable headers
+	const additionalResponseLog = res.headers.get(
```

Comment:
will this ever return anything other than `X-Mf-Additional-Response-Log`? Looking at https://github.com/cloudflare/workers-sdk/pull/8887/files#diff-5a63ebaa83d43a8f47c269b3a4a28584506d7c439860b372b111cc3ecfa6dccaR49 we seem to be setting that header but without a specific value? Is that meant to work as some sort of flag?

Reply:
This is a generic thing. Any Worker can now return an `X-Mf-Additional-Response-Log` header, and it'll get stripped off and logged to console rather than actually be returned as part of the response. We just happen to only have a single user of that for now: asset-worker.